### ISSUE TITLE
provide simpla as a python package

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#10825]
It should be installable and usable via pip which should not be much of a problem since it is pure python. The only question which remains: 

- make a separate package (clearer for the user, separation of concerns) _or_
- include into traci package (less management overhead, it is a small one anyway)

**Comment by namdre** [comment]
isn't it already part of the sumo package?

**Comment by behrisch** [comment]
Yes in the sense that the files are there but it is not easily usable. You cannot do `pip install eclipse-sumo` and then `import simpla` will simply work, you still need to adapt the python path. Furthermore we provide the sumo package only for x86_64 while traci is available "everywhere".
